bciers/apps/registration/app/utils/jsonSchema/facilitiesLfo.ts

```diff
@@ -0,0 +1,78 @@
+import { RJSFSchema } from "@rjsf/utils";
```

Comment:
Do you think it makes sense for `jsonSchema` to live in `app/utils`? Nothing about a schema sounds very util-y to me...I think this is something that happened in early registration1 development for some reason.

Maybe it should live in `/app/data` folder or somewhere else?